docs/LwM2M_Client/STMicroelectronics/Building_LwM2M_applications/Building_LwM2M_applications_with_I-CUBE-Anjay.md

```diff
@@ -113,7 +113,7 @@ Through Select Components menu choose desired components from the pack, in this
 - Board Support LwM2M
     - LwM2M Objects: P-L496G-CELL02
     - P-L496G-CELL02 BSP
-- Board Support X_STMOD_PLUS_MODEMS - BG96 or MONARCH (depending on your setup)
```

Comment:
Actually I-CUBE-Anjay still supports MONARCH modem and will support it for a while, so there is no use in deleting it here.